apps/src/templates/sectionProgress/progressTables/progressTableStyles.scss

```diff
@@ -63,7 +63,7 @@ $content-view-width: $content-width - $student-list-width;
       width: $student-list-width;
     }
     .progress-table-student-name {
-      width: $student-list-width;
+      width: $student-list-width - 4px;
```

Comment:
The parents were set to  $student-list-width, but I think the borders of parents made the width available for this child less than that full width. This was the quickest/easiest fix I could think of without getting into the weeds of how borders effect widths in table elements